wordle-web/src/views/WordleView.vue

```diff
@@ -1,21 +1,18 @@
 <template>
   <h1>Wordle Mind Bender</h1>
-
+  <h2>{{ subtitle }}</h2>
   <GameBoard :game="game" @letterClick="addChar" />
+  <br />
```

Comment:
im not sure if the teachers allow it but consider using flex/grid properties to do spacing instead of using <br/> tags since they can clog up the file and are repetitive

Reply:
ikr